src/app/caesar-portal/group-list/group-list.component.scss

```diff
@@ -42,3 +42,32 @@
     width: 100%;
   }
 }
+
+// circle btn 
+.btn-circle {
+  width: 30px;
+  height: 30px;
+  text-align: center;
+  padding: 6px 0;
+  font-size: 12px;
+  line-height: 1.428571429;
```

Comment:
copy paste from bootstrap 3 examples , seems it does not impact on styles , so removed